lib/opentelemetry_phoenix.ex

```diff
@@ -97,11 +75,12 @@ defmodule OpentelemetryPhoenix do
   end
 
   @doc false
-  def handle_endpoint_start(_event, _measurements, %{conn: %{adapter: adapter} = conn}, _config) do
+  def handle_router_dispatch_start(_event, _measurements, %{conn: conn} = meta, _config) do
     # TODO: maybe add config for what paths are traced? Via sampler?
     :otel_propagator.text_map_extract(conn.req_headers)
 
-    span_name = "HTTP #{conn.method}"
+    span_name = "#{conn.method} #{meta.route}"
```

Comment:
Is `meta.route` here sanitized or anything, e.g. placeholders instead of real values?

Reply:
yep it would return `/v1/merchants/:id`, for example